components/org.wso2.carbon.identity.sso.agent.saml/src/main/java/org/wso2/carbon/identity/sso/agent/saml/bean/SSOAgentConfig.java

```diff
@@ -296,6 +307,12 @@ public void initConfig(Properties properties) throws SSOAgentException {
                 skipURIs.add(skipURI);
             }
         }
+        if (!StringUtils.isBlank(properties.getProperty(SSOAgentConstants.SSOAgentConfig.INDEX_PAGE))) {
+            setIndexPage(properties.getProperty(SSOAgentConstants.SSOAgentConfig.INDEX_PAGE));
+            skipURIs.add(indexPage);
+        } else {
+            setIndexPage("samlsso?SAML2.HTTPBinding=HTTP-POST");
```

Comment:
samlsso endpoint will not be a static endpoint as it will be determined by the web.xml. Same thing we have to add as a properties file, so let's use that config, rather hardcoding the endpoint. Also we can have another config to select preferred binding type and use that to inherit value for query param.

Reply:
Thanks. This is now resolved.